packages/core/src/components/essentials/legend.ts

```diff
@@ -64,6 +64,13 @@ export class Legend extends Component {
 			"numCharacter"
 		);
 
+		const paletteIndex = Tools.getProperty(
+			options,
+			"color",
+			"paring",
```

Comment:
```suggestion
			"pairing",
```

not sure im fully for the naming "pairing" but either way this is the right spelling